pkg/client/postgres/tasks.go

```diff
@@ -95,9 +98,40 @@ func (db *PostgresDB) ScheduleBackup(schedule string) error {
 	return db.client.Apply(db.Namespace, job)
 }
 
-func (db *PostgresDB) ListBackups() ([]string, error) {
-	var backups []string
-	return backups, nil
+func (db *PostgresDB) ListBackups(quiet bool, num int) error {
+	op := db.op.Configuration
+
+	objs := listObjects(db.s3, op.LogicalBackup.S3Bucket, db.Name)
+
+	if !quiet {
+		fmt.Printf("\nURL\tAGE\n")
```

Comment:
Can we add in size?

This should also use a tabwriter rather than inline tabs which are not going to handle differences in value length